markdown/troubleshooting.md

```diff
@@ -18,6 +18,40 @@ Make sure your port and url are both correct in your endpoint. The format should
 
 If you get `No suitable driver found for <connection string>` from the `DriverManager.getConnection` call, then you have an issue with your connection string. This is likely due to an issue in the beginning of the connection string, double check that your connection string starts with `jdbc:neptune:opencypher://`. 
 
+### Maven Issues
+
+If you are using the regular JDBC driver (not shadow) jar as a dependency for your Maven project then you may encounter some errors with transitive dependencies. If you notice exceptions like `java.lang.ClassNotFoundException: com.fasterxml.jackson.core.util.JacksonFeature` when running your application then you probably have a version conflict with some of the dependencies. You will have to add an exclusion for that particular artifact and include the correct version yourself. Since the JDBC driver is built using Gradle, you can emulate how Gradle resolves dependencies and include the highest version from the dependency graph. Using jackson as an example, you can add the following to your `pom` file.
+
+```
```

Comment:
```suggestion
The example below shows how to exclude Jackson Library in case of conflicts and include the version that is required in the project.

```
```